Providers/Profile.swift

```diff
@@ -231,6 +227,7 @@ open class BrowserProfile: Profile {
         // Set up our database handles.
         self.loginsDB = BrowserDB(filename: "logins.db", secretKey: BrowserProfile.loginsKey, schema: LoginsSchema(), files: files)
         self.db = BrowserDB(filename: "browser.db", schema: BrowserSchema(), files: files)
+        self.readingListDB = BrowserDB(filename: "ReadingList.db", schema: ReadingListSchema(), files: files)
```

Comment:
I wish we didn't have to open this for users who don't use RL :/

Reply:
We don't actually open the DB or do anything with the schema until RL is first accessed :-)